### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by Mutai-Gilbert**
In this pull request I was able to:
- Use React components.
- Use React props.
- Use React state.

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <span className="button">2</span>
+    <span className="button">3</span>
+    <span className="button">+</span>
+    <span className="button zero">0</span>
+    <span className="button">.</span>
+    <span className="button orange">=</span>
+  </div>
+);
+
+class Calculator extends React.Component {
```
Comment (TedLivist) [inline src/components/calculator.js:29]: - [ ] This is a class component, we should refactor to a functional component instead

File: src/components/calculator.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <span className="button">&minus;</span>
+    <span className="button">1</span>
+    <span className="button">2</span>
+    <span className="button">3</span>
+    <span className="button">+</span>
+    <span className="button zero">0</span>
+    <span className="button">.</span>
+    <span className="button orange">=</span>
+  </div>
+);
```
Comment (TedLivist) [inline src/components/calculator.js:27]: - [ ] Kindly move this Buttons component to its own file.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Mutai-Gilbert,

Good job so far!
✔️ Use of child components
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [x] Kindly delete these boilerplate files.

  mjd

_Check the comments under the review._

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
## Hello @Mutai-Gilbert 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Kindly make sure to remove all comments and instructions of the README must be filled in order to keep your repo clean and professional.

- [OPTIONAL] Kindly make sure next time to give your branch a descriptive name. (for example: `react-components`).

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**